packages/gatsby-plugin-offline/README.md

```diff
@@ -68,7 +68,14 @@ const options = {
 
 If you want to remove `gatsby-plugin-offline` from your site at a later point,
 substitute it with [`gatsby-plugin-remove-serviceworker`](https://www.npmjs.com/package/gatsby-plugin-remove-serviceworker)
-to safely remove the service worker:
+to safely remove the service worker. First, install the new package:
+
+```bash
+yarn add gatsby-plugin-remove-serviceworker
```

Comment:
We recommend using `npm` (and folks who _would use yarn_ would already know they can swap the command).

Reply:
Okay, I've updated to use NPM!